playground/frontend/playground_components/lib/widgets/split_view.dart

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
What existing packages did you try for split view?
Below applies only if we don't find a suitable one.

Reply:
Agreed to use this one as we already have it.